extensions/shared/external-media/sources/with-media.js

```diff
@@ -186,14 +186,27 @@ export default function withMedia() {
 					path: apiUrl,
 					method: 'POST',
 					data: {
+						external_ids: items.map( item => item.guid ), // WPCOM.
 						media: items.map( item => ( {
 							guid: item.guid,
 							caption: item.caption,
 							title: item.title,
 						} ) ),
+						service: source, // WPCOM.
 					},
 				} )
 					.then( result => {
+						// Convert response on Simple Sites.
+						if ( result.media ) {
```

Comment:
```suggestion
						if ( result?.media ) {
```

Reply:
Can you provide more context for this suggestion? Does it cause an error without it? Is there a way for me to reproduce `result` being empty?

Reply:
Sorry, I submitted this suggestion on accident. I had started to look at this PR shortly before our retro/demo on Thu and put this suggestion as note to myself when initially skimming the code. I switched to trying to functionally test this and then ran into an issue with applying the patch (patch update needed). When the meeting came up and I closed the tab, I must have accidentally clicked "Add comment" instead of "Cancel". I don't think we need this here given that we know result will be present.